package_test.go

```diff
@@ -47,32 +47,32 @@ func testPackage(ctx context.Context, t *testing.T, client *dagger.Client, spec
 		t.Fatalf("unknown distro: %s", spec.Distro)
 	}
 
-	buildOutput, err := do(ctx, client, spec)
+	buildOutput, err := do(ctx, client.Pipeline("Build "+spec.Pkg+" for testing"), spec)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	batsCore, batsHelpers := makeBats(client)
 
-	qemu := testutil.NewQemuImg(ctx, client)
+	qemu := testutil.NewQemuImg(ctx, client.Pipeline("Qemu"))
```

Comment:
Are these `Pipeline` arguments just for logging?

Reply:
Yes, and with the expirmental dagger TUI it helps *a lot*.